tests/test_verifyStats.py

```diff
@@ -0,0 +1,264 @@
+        mi = self.inputExp.getMaskedImage()
+        mi.scaledMinus(1.0, fakeBias.getMaskedImage())
+        makeMockExp(self.inputExp)
+
+        self.camera = isrMock.IsrMock().getCamera()
+
+    def test_bias(self):
+        # Test a subset of the output values to identify that the
+        # image stat methods haven't changed.
+        task = cpVerify.CpVerifyBiasTask()
+        results = task.run(self.inputExp, self.camera)
+        resultStats = results.outputStats
+
+        self.assertAlmostEqual(resultStats['AMP']['C:0,0']['MEAN'], 2.08672, 4)
+        self.assertAlmostEqual(resultStats['AMP']['C:0,0']['NOISE'], 13.99547, 4)
+        self.assertAlmostEqual(resultStats['AMP']['C:0,0']['CR_NOISE'], 14.11526, 4)
+
+        self.assertIn(resultStats['SUCCESS'], [True, False])
+
+
+class VerifyDarkTestCase(lsst.utils.tests.TestCase):
+    """Unit test for stats code.
+    """
+
+    def setUp(self):
+        # Generate a mock exposure/camera to test.
+        config = isrMock.IsrMockConfig()
+        config.isTrimmed = True
+        config.rngSeed = 12345
+        darkExposure = isrMock.DarkMock(config=config).run()
+
+        config.rngSeed = 54321
+        fakeDark = isrMock.DarkMock(config=config).run()
+
+        self.inputExp = darkExposure.clone()
+        mi = self.inputExp.getMaskedImage()
+        mi.scaledMinus(1.0, fakeDark.getMaskedImage())
+        makeMockExp(self.inputExp)
+
+        self.camera = isrMock.IsrMock().getCamera()
+
+    def test_dark(self):
+        # Test a subset of the output values to identify that the
+        # image stat methods haven't changed.
+        task = cpVerify.CpVerifyDarkTask()
+        results = task.run(self.inputExp, self.camera)
+        resultStats = results.outputStats
+
+        self.assertAlmostEqual(resultStats['AMP']['C:0,0']['MEAN'], 2.0043, 4)
+        self.assertAlmostEqual(resultStats['AMP']['C:0,0']['NOISE'], 3.12948, 4)
+        self.assertAlmostEqual(resultStats['AMP']['C:0,0']['CR_NOISE'], 3.15946, 4)
+
+        self.assertIn(resultStats['SUCCESS'], [True, False])
+
+
+class VerifyDefectsTestCase(lsst.utils.tests.TestCase):
+    """Unit test for stats code.
+    """
+
+    def setUp(self):
+        # Generate a mock exposure/camera to test.
+        config = isrMock.IsrMockConfig()
+        config.isTrimmed = True
+        config.doGenerateImage = True
+        config.doAddFringe = False
+        config.doAddSource = False
+        config.doAddSky = True
+        config.doAddOverscan = False
+        config.doAddCrosstalk = False
+        config.doAddBias = False
+        config.doAddDark = False
+        config.doAddFlat = False
+        config.doAddFringe = False
+
+        config.skyLevel = 1000
+        config.rngSeed = 12345
+        self.inputExp = isrMock.IsrMock(config=config).run()
+
+        # These are simulated defects
+        self.inputExp.getImage().getArray()[0, 0] = -100000
+        self.inputExp.getImage().getArray()[40, 50] = 100000
```

Comment:
Since you are testing against these numbers below, perhaps define a variable?

Reply:
I've added a class variable so the generation and testing match.